Sources/HaCWebsiteLib/Events/EventServer.swift

```diff
@@ -2,22 +2,25 @@ import Foundation
 
 public struct EventServer {
 
-  static func getEvents() -> [GeneralEvent] {
-    //TODO: source events from database
-    let hypeDuration : Double = 4000000
-    let duration : Double = 4000
-    let workshopEvents : [GeneralEvent] = WorkshopManager.workshops.map{
-      (workshop: Workshop) -> GeneralEvent in
-        return GeneralEvent(title: workshop.title, time:  DateInterval(start: Date(), duration: duration),
-          tagLine: "There's Pizza", color: "purple", hypePeriod: DateInterval(start: Date(), duration: hypeDuration),
-          tags: ["tag1", "tag2"], description: Markdown("Thing"))
-    }.filter{ event in 
-      event.shouldShowAsUpdate
+  private(set) static var events: [GeneralEvent] = []
+
+  static func update() {
+    let newEvents = try? GeneralEvent.makeQuery().filter("title", .notEquals, nil).all()
```

Comment:
When would a title be nil? In GeneralEvent, this is non-optional

Reply:
Never - it's just a way of getting all the events from the database as it seems with `Fluent` you have to specify some form of filter :(

Reply:
@CharlieCrisp Probably worth commenting then!

Reply:
`all()` is a method on QueryRepresentable, doesn't look like we need that filter clause at all!